layers/+filetree/treemacs/packages.el

```diff
@@ -41,6 +41,11 @@
             treemacs-never-persist nil
             treemacs-goto-tag-strategy 'refetch-index
             treemacs-collapse-dirs treemacs-use-collapsed-directories)
+      (unless treemacs-lock-width
+        (add-hook 'treemacs-mode-hook
+                  (lambda ()
```

Comment:
Just a thought. Maybe lambda is not the best thing to place into a hook. But it's minor.

Reply:
Would you prefer a named function or do you have something else in mind ?